nexusdialog/src/main/java/com/github/dkharrat/nexusdialog/validations/ValidationErrorDisplayMethod.java

```diff
@@ -0,0 +1,53 @@
+package com.github.dkharrat.nexusdialog.validations;
+
+import android.content.Context;
+import android.content.res.Resources;
+
+import com.github.dkharrat.nexusdialog.FormController;
+import com.github.dkharrat.nexusdialog.FormElementController;
+import com.github.dkharrat.nexusdialog.R;
+import com.github.dkharrat.nexusdialog.utils.MessageUtil;
+
+/**
+ * Defines methods to display the validation errors.
+ * You MUST set the {@link ValidationErrorDisplayMethod#context} and {@link ValidationErrorDisplayMethod#controller}
+ * before calling showErrors.
+ */
+public enum ValidationErrorDisplayMethod {
```

Comment:
Instead of an enum, I believe it would be better to use an interface and provide different implementations. That way, an app can easily extend NexusDialog to support other display methods. An enum is not extensible without modifying NexusDialog itself.